.github/workflows/code-quality.yml

```diff
@@ -58,6 +58,9 @@ jobs:
       - name: Install Dependencies
         run: pnpm install
 
+      - name: Install eslint cli
+        run: pnpm install -w eslint
+
```

Comment:
I am not sure why, but linting was failing with `eslint command not found`.  This is the easiest way I could find to get CI passing.

Reply:
created a follow up to make sure we properly fix the eslint problem: https://github.com/recallnet/js-recall/issues/318